public/walkthroughs/1/walkthrough.adoc

```diff
@@ -0,0 +1,345 @@
+.. Select *JSON Schema* as the type.
+.. Enter the following in the *Definition* field:
++
+[subs="attributes+"]
+----
+{
+	"$schema": "http://json-schema.org/draft-04/schema#",
+	"type": "object",
+	"properties": {
+		"type": {
+			"type": "string"
+		}
+	}
+}
+----
+.. Click *Done*.
+
+. Choose *Walkthrough {walkthrough-id} CRUD App* as the *Finish Connection*.
+
+. When prompted to *Choose an Action*, select *Create a fruit*.
+
+. When prompted to *Add to Integration*, select *Add a Step*.
+
+. Select *Data Mapper* to map source and target fields in the corresponding JSON schemas:
+.. Click and drag *type* from the *Source* column to *body/name* in the *Target* column.
+.. Click *Done* to navigate back to the Integration screen.
+
+. Click *Publish*.
+. When prompted, enter a name, for example:
++
+[subs="attributes+"]
+----  
+Walkthrough {walkthrough-id}
+----
+
+[type=verification]
+Check the *Integrations* screen of the link:{fuse-url}[Red Hat Fuse Online, window="_blank"] console to make sure the new connection exists.
+
+[type=verificationFail]
+Verify that you followed each step in the procedure above.  If you are still having issues, contact your administrator.
+// end::task-creating-fuse-integration[]
+
+
+[type=taskResource]
+.Additional Information
+****
+* link:https://access.redhat.com/documentation/en-us/red_hat_fuse/7.0/[Documentation]
+* link:https://www.redhat.com/en/technologies/jboss-middleware/fuse[About Fuse]
+****
+
+[time=5]
+[id='using-the-application-integration']
+== Using the application integration
+:context: using-the-application-integration
+
+// tag::task-using-integration[]
+[id='using-integration_{context}']
+[.integr8ly-docs-header]
+=== Using the application integration
+
+After setting up the integration between the Node.js and Spring Boot applications, we can use the integration to create a new fruit for the grocery inventory application.
+
+:node-url: http://frontend-node-app.apps.city.openshiftworkshop.com/
+:spring-url: http://spring-boot-rest-http-crud-spring-app.apps.city.openshiftworkshop.com/
+:fuse-url: https://eval.apps.city.openshiftworkshop.com/
+
+. Navigate to the link:http://{node-js-url}[Node.js app, window="_blank"].
+
+. Enter a value for *Fruit*, for example:
++
+----  
+Pineapple
+----
+
+. Click *Send Request*.
+
+. Navigate to the link:http://{spring-boot-url}[Spring Boot app, window="_blank"].
+
+. Check that the entry from the Node.js app is displayed.
+
+
+[type=verification]
+****
+View the activity log:
+
+. Log in to the link:{fuse-url}[Red Hat Fuse Online, window="_blank"] console.
+. Select *Integrations* from the left hand menu.
+. Select your integration.
+. Select the *Activity* tab.
+. Expand the log entry to display the steps performed.
+****
+
+[type=verificationFail]
+Verify that you followed each step in the procedure above.  If you are still having issues, contact your administrator.
+
+
+// end::task-using-integration[]
+
+[type=taskResource]
```

Comment:
For some reason this gets picked up as a separate task instead of resources on the right hand side:

![bildschirmfoto 2018-11-22 um 14 50 06](https://user-images.githubusercontent.com/1851198/48906968-76159180-ee66-11e8-92ec-ebd84bb7f372.png)